src/test/java/org/minions/devfund/katerinaanzoleaga/battleship/OceanTest.java

```diff
@@ -0,0 +1,95 @@
+package org.minions.devfund.katerinaanzoleaga.battleship;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * Ocean test class.
+ */
+public class OceanTest {
+    private Ocean theOcean;
+    private BattleShip battleShip = new BattleShip();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "battleShip" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)